### ISSUE TITLE
valid loss of audio2face

### ISSUE THREAD

**Issue by stupiding** [zhangchenxu528/FACIAL#69]
I noticed that the finetuning of audio2face model just use all the data as train set. 
When split the data into train and valid, I found **the loss of generator overfitted**. 
And another worth-noting thing is, the **variation of audio2face's prediction is much smaller** than ground truth.
So is there anything we can do to alleviate the overfit, or to enlarge prediction's variantion?

**Comment by zhangchenxu528** [comment]
Good questions. For the overfit, you may decrease the learning rate or use the early stop. 

The variation will be reduced if the length of the finetuning video is too long. You may increase the step in dataset102. Now, it is set to 1. (list(np.arange(0, min_num_frame-self.frames, 1) + base))
